src/nyc_trees/apps/event/views.py

```diff
@@ -23,8 +23,9 @@
 
 
 def event_dashboard(request):
-    # TODO: implement
-    pass
+    base_url = reverse('group_detail',
+                       kwargs={'group_slug': request.group.slug})
+    return HttpResponseRedirect(base_url + '#group-events')
```

Comment:
I don't think this is quite right.

The original idea of the `event_dashboard` from the proposal is that it is the place for admins to manage events, and I believe it has evolved into the group tab of the event management page.
This is why the route has a `group_admin_do` decorator on it.

It makes sense for "Find more events" link to go to the events section of the publicly visible group detail, but the events section is different from the group admin's event dashboard.